src/layer/AbstractDEMLayer.ts

```diff
@@ -71,13 +71,13 @@ export class AbstractDEMLayer extends AbstractSentinelHubV3Layer {
   }
 
   public async getMap(params: GetMapParams, api: ApiType, reqConfig?: RequestConfiguration): Promise<Blob> {
-    const getMapValue = await ensureTimeout(async innerReqConfig => {
+    params = await this.decideJpegOrPng(params);
```

Comment:
Shouldn't this also be inside the timeout?

Reply:
Good point - yes, it should be, and it should also pass `innerReqConfig` along, to make sure timeout is observed. Thank you for pointing this out! Fixed.